interactive_ai/supported_models/geti_supported_models/manifests/detection/atss_mobilenetv2.yaml

```diff
@@ -16,3 +16,4 @@ supported_gpus:
 hyperparameters:
   training:
     learning_rate: 0.004
+    default_input_size: "800x992"
```

Comment:
I think it should be:
```yaml
input_size_width: 800
input_size_height: 992
```
@maxxgx correct me if I'm wrong

Reply:
Done for all tasks